game-core/src/main/java/games/strategy/triplea/attachments/AbstractUserActionAttachment.java

```diff
@@ -25,8 +28,10 @@
 
   // a key referring to politicaltexts.properties or other .properties for all the UI messages belonging to this action.
   protected String m_text = "";
-  // cost in PU to attempt this action
+  @Deprecated
```

Comment:
I'm curious what should be done about the deprecated usages. Would you mind putting a code comment what should be done, should we stop using the variable altogether, or use something else perhaps?

Reply:
costResources should now always be used instead of costPU as its a superset. I can add a comment, guess I just figured it was obvious :)